src/components/textarea/Textarea.tsx

```diff
@@ -74,6 +74,13 @@ const StyledTextarea = styled.textarea<TextareaType>`
   &.has-label {
     padding: ${theme.space24} 4px ${theme.space8} ${theme.space16};
 
+    ::placeholder {
+      transition: ${theme.transition};
+      opacity: 0;
+      color: ${theme.gray400};
+      font-weight: 400;
+    }
+
```

Comment:
Now I'm thinking more about this. What do you think about having `::placeholder` outside the scope of `&.has-label` with css like `color`, `transition` and `font-weight`. And here you would have just `opaicity: 0`? So that in the regular `textarea` with no `label`, the placeholder style would stay the same.
```suggestion
    ::placeholder {
      opacity: 0;
    }

```